README.md

```diff
@@ -1 +1,26 @@
-# github-flow
\ No newline at end of file
+# github-flow
+
+Brief description of the project.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Installation
+
+Instructions on how to install and run the project.
+
+## Usage
+
+Instructions on how to use the project.
+
+## Contributing
```

Comment:
As the previous reviewer suggested, In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.
Kindly add all sections from the Microverse README and ensure to customize it to be in line with the project repository 👍 . You can refer to this [repository](https://github.com/GhostEsso/Hello-Microverse/tree/htmlcss-branch) to customize your README.
The only optional section from the README file is faq.

Reply:
Thank you very much